components/Message.jsx

```diff
@@ -10,11 +9,9 @@ const Message = React.createClass({
   render() {
     // Destructuring pulls the different fields from
     // the message object
-    const {name, message} = this.props.message;
```

Comment:
would leave this line unchanged and same with the `emojifiedString` part to match the slides